### PR TITLE
Minimal changes to combine newflow and old flow

### PR DESCRIPTION
**Pull request by Dantemss**
Still need to fix specs and combine styles/layout and OAuth providers

### CODE REVIEW

File: app/controllers/faculty_access_controller.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class FacultyAccessController < ApplicationController
```
Comment (mwvolo) [inline app/controllers/faculty_access_controller.rb:1]: @Dantemss not sure there is still a path to this controller - I think the faculty access application process is handled in `educator_signup_controller`

File: app/mailers/sign_in_help_mailer.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 class SignInHelpMailer < ApplicationMailer
+  include Rails.application.routes.url_helpers
```
[the remark this answers is not in the document]
Comment (Dantemss) [reply app/mailers/sign_in_help_mailer.rb:2]: I'm not 100% sure mailers need this, but generally non-controller files that need access to the url helpers

**Comment by Dantemss** [comment]
The only failure I still get locally is the specs that admins get logged out after 30 min, but only when running multiple specs in a row, oddly enough.

File: config/routes.rb [hunk 3]
```diff
@@ -76,7 +62,7 @@
 
     # Step 3
     get 'signup/educator/apply', action: :educator_sheerid_form, as: :educator_sheerid_form
-    post 'i/sheerid/webhook', action: :sheerid_webhook, as: :sheerid_webhook
+    post 'sheerid/webhook', action: :sheerid_webhook, as: :sheerid_webhook
```
Comment (mwvolo) [inline config/routes.rb:65]: the `/i/` in this post path is hard-coded in the sheer id platform and might be difficult to change (no, it is... I check with them).
Is there some nginx magic to handle this?

File: spec/support/feature_helpers.rb [hunk 4]
```diff
@@ -245,10 +252,12 @@ def complete_login_password_screen(password)
 end
 
 def complete_reset_password_screen(password=nil)
+  expect(page.current_path).to eq(password_reset_path)
   password ||= 'Passw0rd!'
-  fill_in (t :"identities.set.password"), with: password
-  fill_in (t :"identities.set.confirm_password"), with: password
+  fill_in 'set_password_password', with: password
```
Comment (mwvolo) [inline spec/support/feature_helpers.rb:257]: just so I know - what are you using as the selector here? The form field name?

**Review by mwvolo — APPROVED** [review]

File: spec/support/feature_helpers.rb (hunk 4 above)
Comment (Dantemss) [reply spec/support/feature_helpers.rb:257]: The form field id

File: config/routes.rb (hunk 3 above)
Comment (Dantemss) [reply config/routes.rb:65]: We can add a rewrite I think

**Pull request merged**